mffpy/tests/test_reader.py

```diff
@@ -95,6 +101,49 @@ def test_get_physical_samples_full_range(reader):
     reader.get_physical_samples_from_epoch(reader.epochs[0])
 
 
+@pytest.mark.parametrize("t0,expected_eeg,expected_pns,expected_start", [
+    (0.0, [
+        -1845.62, -1844.3558, -1844.1255, -1844.5186, -1845.0215, -1844.4167,
+        -1843.2725, -1843.3767, -1843.8182, -1844.8408, -1843.2706, -1843.5922
+    ], [
+        0.21565743, -0.51517457, -1.3356407, -0.16099238, 1.4178662, 0.7046892,
+        -0.66320664, -0.09752636, 0.72956467, -0.04039904, -0.14828575,
+        1.3500004
+    ], 0.0),
+    (0.01, [
+        -1844.1255, -1844.5186, -1845.0215, -1844.4167, -1843.2725, -1843.3767,
+        -1843.8182, -1844.8408, -1843.2706, -1843.5922, -1845.4554, -1844.5149,
+        -1843.0234
+    ], [
+        -1.3356407, -0.16099238, 1.4178662, 0.7046892, -0.66320664,
+        -0.09752636, 0.72956467, -0.04039904, -0.14828575, 1.3500004,
+        1.0464033, -1.5411711, -1.9540663
+    ], 0.008),
+    (0.02, [
+        -1844.4167, -1843.2725, -1843.3767, -1843.8182, -1844.8408, -1843.2706,
+        -1843.5922, -1845.4554, -1844.5149, -1843.0234, -1843.368, -1844.1075,
+        -1844.739
+    ], [
+        0.7046892, -0.66320664, -0.097526364, 0.72956467, -0.040399045,
+        -0.14828575, 1.3500004, 1.0464033, -1.5411711, -1.9540663, 0.6091788,
+        1.8593298, 1.2873881
+    ], 0.02),
+])
+def test_get_physical_samples_multiple_bins(t0, expected_eeg, expected_pns,
+                                            expected_start, mffpath_3):
+    expected_eeg = np.asarray(expected_eeg, dtype=np.float32)
```

Comment:
Fundamentally - this is a good test, but it would also be good to add a function comment here, so another programmer will understand what each of these variables point to, and precisely the aims of your test here.